Update http remote documentation (#1767)
iterative/dvc.org: content/docs/command-reference/remote/modify.md

  $ dvc remote modify myremote url https://example.com/path/to/dir
  ```

>   > The url can include query parameters that will be preserved.

Reviewer: ```suggestion
  > The URL can include a query string, which will be preserved (e.g.
  > `example.com?loc=path%2Fto%2Fdir`)
```

  > `password` is specified, DVC will not prompt the user to enter a password
  > for this remote.

> - `ssl_verify` - allows to disable ssl verification. It is enabled by default.

Reviewer: ```suggestion
- `ssl_verify` - allows to disable SSH verification, which is enabled by default.
```

> - `ssl_verify` - allows to disable SSH verification, which is enabled by default.

Reviewer: ```suggestion
- `ssl_verify` - allows to disable SSH verification, which is enabled by
  default.
```
